crates/anvil/tests/it/anvil_api.rs

```diff
@@ -792,4 +792,8 @@ async fn test_reorg() {
         })
         .await;
     assert!(res.is_err());
+
+    if let Some(signal) = handle.shutdown_signal_mut().take() {
+        signal.fire().unwrap();
+    }
```

Comment:
could we convert this to a function on handle?

and document why we call this for some tests